faces.py

```diff
@@ -156,6 +156,23 @@ def get_random_in_batch(cursor, current_user):
                    [current_user,current_user])
     return [x for x in cursor.fetchall()]
 
+def get_stints_for_user(cursor, user):
+    """Returns the stint type, name, and duration for each stint in which the current
+    user participated."""
+    cursor.execute("""SELECT
+                        stints.stint_type,
+                        stints.start_date,
+                        stints.end_date,
+                        stints.title,
+                        batches.short_name
+                      FROM stints
+                      LEFT JOIN batches
+                        ON stints.batch_id = batches.batch_id
+                      WHERE stints.person_id = %s
+                      ORDER BY batches.batch_id""",
```

Comment:
I wanted them to appear in chronological order, so this was an oversight! I'll make that change to use `stints.start_date`.

We refactored this previously to use `batches.batch_id` instead of `batches.start_date` when `start_date` was removed from `batches` as part of #48. It was safe to use `batches.batch_id` when I was only displaying info for RCers, but I should have updated this after accepting the stretch goal to display stint info for everyone. My mistake, but thanks for catching it!